README.md

```diff
@@ -70,6 +74,14 @@ jobs:
         version: ${{ github.event.release.tag_name }}
         zenodo_json: .zenodo.json
         archive: ${{ env.archive }}
+
+        # Optional DOI for all versions. Leaving this blank (the default) will create
+        # a new DOI on every release. Use a DOI that represents all versions will
+        # create a new version for this existing DOI.
+        #
+        # Newer versions have their own DOIs, but they're also linked to this DOI
+        # as a different version. When using this, use the DOI for all versions.
+        doi: '10.5281/zenodo.6326823'
```

Comment:
```suggestion
        doi: '10.5281/zenodo.6323747'
```

Should this example a doi for all versions? I have this one handy.

Reply:
Wouldn't the token in the actions (that I created) need ownership over it?

Reply:
Yea I'd just meant to use it now as an example.

I guess what we'd *actually* want is 2 release actions - one for each variant? So we have examples for both.

Reply:
That would work! You'd want to have the token you need in actions - let me know if you want maintainer on the repo and I can add you to do that.

Reply:
This example is now `10.5281/zenodo.6326822` the DOI for all versions of 0.0.15.